core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -629,9 +639,12 @@ class QualificationAppInfo(
       val totalTransitionsTime = allStagesSummary.map(s => s.transitionTime).sum
       val unsupportedSQLTaskDuration = calculateSQLUnsupportedTaskDuration(allStagesSummary)
       val endDurationEstimated = this.appEndTime.isEmpty && appDuration > 0
-      val jobOverheadTime = calculateJobOverHeadTime(info.startTime)
+      val endTimeJob = info.endTime.getOrElse(0L)
+      val jobOverheadTime = calculateJobOverHeadTime(info.startTime, endTimeJob)
       val nonSQLDataframeTaskDuration =
         calculateNonSQLTaskDataframeDuration(sqlDataframeTaskDuration, totalTransitionsTime)
+
+      // this is weird we are adding wallclock with serialized task duration
       val nonSQLTaskDuration = nonSQLDataframeTaskDuration + jobOverheadTime
```

Comment:
@tgravescs @amahussein - This isn't correct as we are adding task duration to wallclock duration(job overheadtime).

This is used in output file as  "NONSQL Task Duration Plus Overhead" which is also not right. Wanted your opinion if it would be good to convert this into wallclock duration instead? 
In this PR, we are not using nonSQLTaskDuration for estimating the speedup.

Reply:
eventually we need to just separate them out so someone looking at results can see them separately.  Lets do that under a separate issue - please file one